common/app/components/VideoPlayer.tsx

```diff
@@ -164,7 +164,9 @@ const showingSubtitleHtml = (
 `;
     }
 
-    return `<span style="${subtitleStyles}" class="${subtitleClasses}">${subtitle.text}</span>`;
+    const lines = subtitle.text.split('\n');
```

Comment:
This change should be repeated on line 198 to get the same behavior when DOM caching is disabled.

Reply:
![image](https://github.com/user-attachments/assets/6be68999-5ca3-45f5-87f4-1417f0092aac)
![image](https://github.com/user-attachments/assets/7aa2bb68-71a8-4603-a00a-48c6c8a3e01c)